README.md

```diff
@@ -0,0 +1,239 @@
+
+## Using the Studio to create Polls and Surveys
+
+After adding a Poll or Survey to the unit you're editing, click the edit button to bring up the studio menu.
+
+Polls will have a `Question` field, and both polls and surveys will have a `Feedback` field.
+
+![Poll edit screen](doc_img/poll_edit.png)
+
+After these fields, there will be a series of line items. Polls will have a set of `Answers`:
+
+![Poll line items](doc_img/poll_line_items.png)
+
+...while Surveys will have both `Questions` and `Answers`:
+
+![Survey line items](doc_img/survey_line_items.png)
+
+In polls, answers may be given images, while in surveys, questions may be given them.
+See the examples sections to get an idea of how images look and fit into the layout of a poll or survey.
+
+Questions and answers may be enhanced with Markdown and HTML. **This markup is not sanitized, and is treated as trusted
+input by course authors.**
+
+For example, putting this into a Poll's Answer field:
+
+![Markdown mixed with HTML](doc_img/mixed_markdown.png)
+
+...Would yield this when rendered:
+
+![Rendered Markdown mixed with HTML](doc_img/mixed_markdown_render.png)
+
+The following fields are customizable with [Markdown](http://daringfireball.net/projects/markdown/syntax) and custom HTML on Polls:
+
+* Question
+* Feedback
+* Answers
+
+The following fields are customizable with Markdown and custom HTML on Surveys:
+
+* Feedback
+* Questions
+
+You may add another answer (or question, if creating a survey) by clicking the add links at the bottom of the form:
+
+![Add links](doc_img/add_links.png)
+
+Questions and answers are listed in the order they're used in the student display. If you need to change the order
+of questions or answers, **do not swap the values of their fields** if any students may already have voted, as
+those votes will be tied to that line item. Instead, use the ordering arrows on the side:
+
+![Ordering Arrows](doc_img/ordering_arrows.png)
+
+These will allow you to move the question or answer up and down to change its display order.
+
+If you need to remove a question or answer, you can use the `Delete` link:
+
+![Delete Link](doc_img/delete_link.png)
+
+**Remember**: You must have at least two answers (and at least two questions, in the case of polls). If a user has
+voted on/for the item you've deleted, they will be permitted to vote again, but will not lose progress.
+
+You may also create a poll or survey in which the results are not shown to the user. To do this, click the checkbox
+for 'Private Results':
+
+![Private Results](doc_img/private_results.png)
+
+**Notes on Private Results**: Users will be able to change their vote on polls and surveys with this option enabled.
+An analytics event will not be fired upon the student viewing the results, as the results are never visible. A user
+will see a thank you message and any feedback provided upon submission:
+
+![Private Results Submission](doc_img/private_results_submission.png)
```

Comment:
I'm looking at the sandbox and I don't understand how, as the instructor, I can get the results from my block.

Here is where I am: http://sandbox3.opencraft.com/courses/TestOrg/ImportCourse1/now/courseware/d5365b39b81348fb804bf57feaff4264/25bc82de92884537a7fba02fd31f8bc0/

I am logged in as staff. I looked at Studio and answered the question through all the resubmits, thinking maybe one of those options would show me the results of the poll, but neither works.

Either the UX needs to be clearer, or the README needs to explicitly explain - right here! - how instructors can see the results.

If you can tell me how I can, from the sandbox, see the results of the poll as staff I can help suggest how to word documentation.